### ISSUE TITLE
Why is the README mentioning that the cache should be disabled?

### ISSUE THREAD

**Issue by pfrenssen** [openeuropa/media_avportal#4]
The README contains an entire section about how to disable the page cache but it doesn't explain why this is needed.

Does this mean that the module is not compatible with having the page cache enabled? This cache is very important for websites that do not have a dedicated reverse proxy like Varnish.

Can we clarify what is needed to make the module compatible with page caching? It is usually not very difficult to correctly support caching, D8 offers cache tags and cache contexts for this purpose.

**Comment by brummbar** [comment]
You are correct @pfrenssen , I didn't realise that part was in the README. It's misleading, this module doesn't require caching to be disabled.
